File: 250916/남녀노소 구분짓기/sex-and-age.py
gender = int(input())
age = int(input())

if gender == 0 and age >= 19:
    print("MAN")
elif gender == 0 and age < 19:
    print("BOY")
elif gender == 1 and age >= 19:
    print("WOMAN")
elif gender == 1 and age < 19:
    print("GILE")
else:
    print("올바른 숫자를 입력하세요")

File: 251005/남녀노소 구분짓기/sex-and-age.c
#include <stdio.h>

int main() {
    // Please write your code here.
    int gender, age;
    scanf("%d", &gender);
    scanf("%d", &age);

    if (gender == 0){
        if(age >= 19){
            printf("MAN");
        }
        else{
            printf("BOY");
        }
    }
    else{
        if (age >= 19){
            printf("WOMAN");
        }
        else{
            printf("GIRL");
        }
    }

    return 0;
}